src/snowflake/connector/result_batch.py

```diff
@@ -258,6 +258,16 @@ def uncompressed_size(self) -> Optional[int]:
     def column_names(self) -> List[str]:
         return [col.name for col in self.schema]
 
+    @property
+    def id(self) -> str:
+        """Returns an id for the chunk."""
+        if self._remote_chunk_info:
+            parsed_url = urlparse(self._remote_chunk_info.url)
+            path_parts = parsed_url.path.split("/")
```

Comment:
Tiny bit more efficient
```suggestion
            path_parts = parsed_url.path.rsplit("/", 1)
```